collector/generatorreceiver/topos/demo_scenario_c.yaml

```diff
@@ -215,11 +217,13 @@ topology:
           maxLatencyMillis: 200
     cartservice:
       tagSets:
-        - tags:
```

Comment:
Good question! Nope won't interfere, this section now translates to: `cartservice` will have tags `version: v5` and `region: us-east-1` 100% of the time that `runs_on_azure` is _disabled_. It will have tag `region: North_Central_US` 100% of the time that `runs_on_azure` is _enabled_.

Reply:
Awesome! thanks for the review :)